components/pedigree/resources/src/main/resources/pedigree/pedigreeEditorParameters.js

```diff
@@ -126,6 +126,8 @@ define([], function(){
         hoverboxHighlightColor: "green",
         mainGlowColor: "green",
         secondaryGlowColor: "#556B2F"
+        maxPEDexportContainerHeight: 600,
```

Comment:
indeed, refactoring regression. made a separate commit with it, now should build.